src/app/portlets/dot-edit-content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.ts

```diff
@@ -1,22 +1,94 @@
-import { Component, OnInit, Input, EventEmitter, Output } from '@angular/core';
+import { Component, OnInit, Input, EventEmitter, Output, ViewChild } from '@angular/core';
 import { DotMessageService } from '../../../../api/services/dot-messages-service';
+import { SelectItem, MenuItem, SplitButton, InputSwitch } from 'primeng/primeng';
+import { Workflow } from '../../../../shared/models/workflow/workflow.model';
 
 @Component({
     selector: 'dot-edit-page-toolbar',
     templateUrl: './dot-edit-page-toolbar.component.html',
     styleUrls: ['./dot-edit-page-toolbar.component.scss']
 })
 export class DotEditPageToolbarComponent implements OnInit {
-    @Input() pageTitle: string;
+    @ViewChild('actionsButton') actionsButton: SplitButton;
```

Comment:
I think you use it just in the test, if so then you should find another way to do the test

Reply:
```.componentInstace``` forgot about that.